README.md

```diff
@@ -1,24 +1,7 @@
-# ForkDelta
-[![Chat on Discord](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/MPvAfMa)
-[![Join us on Reddit](https://img.shields.io/badge/reddit-ForkDelta-red.svg)](https://www.reddit.com/r/ForkDelta/)
+# Agorex-io
+[Chat on Discord](https://discord.gg/GVuj6UY)
+[Join us on Reddit](https://www.reddit.com/r/Agorex/)
 
-ForkDelta is a decentralized Ethereum Token Exchange with the most ERC20 listings of any exchange.
+This repository contains the old UI forked from [ForkDelta](https://github.com/forkdelta), and is an artifact from the migration of developers from ForkDelta to Agorex-io.
 
-
-# What is ForkDelta?
-ForkDelta currently acts as an open source, updated interface for EtherDelta's smart contract with an active and public development team. Although ForkDelta started as a fork of EtherDelta, considerable development time has been put towards a new token listing system, order book, API, and front-end UI all of which are at different stages of development and can be tracked on [GitHub](https://github.com/forkdelta/). 
-
-
-# How does ForkDelta Work?
-ForkDelta currently interfaces with EtherDelta's original smart contract (described in more detail [here](https://www.reddit.com/r/EtherDelta/comments/6kdiyl/smart_contract_overview/)) which allows ForkDelta users to trade utilizing EtherDelta's volume.
-
-
-# What is the Roadmap for ForkDelta?
-The ForkDelta Roadmap is maintained at https://forkdelta.github.io/about/
-
-
-# Contribute
-We welcome contributions from the community. You can contribute by improving the code, documentation, or by donating. 
-ForkDelta currently interfaces with EtherDelta's original smart contract which allows ForkDelta users to trade utilizing EtherDelta's volume. However, this means that the trading fees still go to EtherDelta and ForkDelta development costs come entirely out of our pockets. If you are interested in the ForkDelta project and would like to contribute to its future, please consider donating to the following Ethereum address: <a href="https://etherscan.io/address/0x3a2aEdc3B54A99e429ae36637681d4560cE5C05b">0x3a2aEdc3B54A99e429ae36637681d4560cE5C05b</a>
-
-As this is a project by the community, for the community, transactions to and from the donations address will be public and can be viewed on the blockchain. We will also keep a public ledger of how the donations are distributed. That way, donors can see exactly how their donations are helping!
+Until the cooperative makes a formal decision regarding a development roadmap, this repository will remain in Agorex-io's GitHub organization.
```

Comment:
Rather than only speak to roadmap maybe use something like:

> We are working with the community to develop our plans for the project. More information will be posted here soon. For now come chat with us on [our Discord](https://discord.gg/GVuj6UY) and help influence the future of an open, honest, community-run decentralized exchange!